api/src/main/java/liquibase/ext/sqlgenerator/core/ModifyColumnGenerator.java

```diff
@@ -192,12 +192,10 @@ String getPostDataTypeString(Database database) {
 	String getDefaultClause(ColumnConfig column, Database database) {
 		String clause = "";
 		String defaultValue = column.getDefaultValue();
-		if (defaultValue != null) {
-			if (database instanceof MySQLDatabase) {
+		if (defaultValue != null && database instanceof MySQLDatabase){
```

Comment:
There shouldn't be a space before the brace here?

Reply:
There is no space except if you are saying i add space